src/components/Navbar/style.scss

```diff
@@ -198,7 +198,7 @@
 		color: $acm-cobalt !important;
 	}
 
-	@media (max-width: 700px) {
+	@media (max-width: 875px) {
```

Comment:
Note @austinkeil:

This is due to the length of the text `fall general meeting` in the nav-bar. When we shorten it to `fall gm` and add the about committees section, this responsive max-width will likely need to change again.